### PR TITLE
Update CoaTran to v0.0.4

### PR DESCRIPTION
**Pull request by niemasd**
The previous CoaTran recipe was on v0.0.1 and was not autobumping. I am updating the recipe to v0.0.4, with it hopefully able to autobump. Note: I am the CoaTran developer

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by niemasd** [comment]
@BiocondaBot please add label

**Comment by niemasd** [comment]
@bioconda/core This PR is ready to review

**Comment by mencian** [comment]
@BiocondaBot please fetch artifacts

File: recipes/coatran/meta.yaml [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
+    - coatran_inftime -h 2>&1 | grep "CoaTran"
+    - coatran_transtree -h 2>&1 | grep "CoaTran"
 
 extra:
   identifiers:
     - biotools:coatran
+    - doi:10.1101/2020.11.10.377499
   recipe-maintainers:
-    - acaprez
+    - acapraz
```
Comment (mencian) [inline recipes/coatran/meta.yaml:41]:
```suggestion
    - acaprez
```
I think you meant `acaprez`?

**Comment by niemasd** [comment]
@mencian Can we revert the edits you made to the tests? The [original version](https://github.com/bioconda/bioconda-recipes/blob/a48d1943dbe597bc980824588eaf641373fda299/recipes/coatran/meta.yaml#L32-L34) seemed to work correctly:

https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=52917&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=4dddc55b-10bc-50fc-ac68-7ff899b7a030&l=724

whereas the [updated version](https://github.com/bioconda/bioconda-recipes/pull/46308/commits/232c3c4a72b0f86d35b1efda1df24325f41d4911#diff-1e2a623db9500a4d17f0031c14f0ed706e6d8ea810087b8e69c974f5be6e6051R33-R35) seems to break in the tests:

https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=53023&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=4dddc55b-10bc-50fc-ac68-7ff899b7a030&l=704

(unless it was the edit to `build.sh` that broke it?)

**Review by mencian — APPROVED** [review]

**Pull request merged**